content/Assets/Scripts/components/addClassOnScroll/index.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Applies a class when page height has been scrolled
+ */
+const SELECTOR = '.js-add-class-on-scroll';
+const defaultCSS = 'page-scrolled';
+let pageHeight = window.innerHeight;
+
+const scrollBehaviour = () => {
+    document.querySelectorAll(SELECTOR).forEach(($el: Element) => {
```

Comment:
Should we store the result of `document.querySelectorAll(SELECTOR)`? Imagine `scroll` is going to be firing a lot and this is going to perform a DOM lookup every time the function is fired.